src/controllers/media.ts

```diff
@@ -69,8 +70,21 @@ export const getByOsdbHash = asyncHandler(async(req: Request, res: Response) =>
     goofs: osMeta.metadata.goofs,
     trivia: osMeta.metadata.trivia,
     tagline: osMeta.metadata.tagline,
+    genres: osMeta.metadata.genres,
+    actors: Object.values(osMeta.metadata.cast),
   };
 
+  // if we're missing values for genres or actors, attempt to hydrate them from imdbAPI instead of OpenSubtitles
+  if ([newMetadata.genres, newMetadata.actors].some(val => val === undefined || []) && newMetadata.imdbID) {
+    try {
+      const imdbData: any = await imdbAPI.get({ id: newMetadata.imdbID });
+      newMetadata.actors = imdbData.actors.split(', ');
+      newMetadata.genres = imdbData.genres.split(', ');
+    } catch (e) {
+      // ignore error, this shouldn't make the request fail
```

Comment:
We should probably be a bit more smart about this since not all errors are equal and could at least be logged instead of disappearing